ogc-custom-test/src/test/java/org/orbisgis/ogccustomtest/OGC_FilterEncodingTest.java

```diff
@@ -244,17 +244,17 @@ public void TestSpatialFilter() throws JAXBException {
 
 
         //test Equals
-        xml = OGC_TestFilterEncoding.class.getResourceAsStream("filter_Equals.xml");
+        xml = OGC_FilterEncodingTest.class.getResourceAsStream("filter_Equals.xml");
         JAXBElement equalsElement = (JAXBElement) unmarshaller.unmarshal(xml);
         comparisonOps = (FilterType) equalsElement.getValue();
 
         JAXBElement<BBOXType> equals = (JAXBElement<BBOXType>) comparisonOps.getSpatialOps();
 
         Assert.assertEquals("Error : the input file isn't a Filter", equalsElement.getName().getLocalPart(), "Filter");
-        Assert.assertEquals("Error : the filter doesn't have the operator Beyond ", equals.getName().getLocalPart(), "Equals");
+        Assert.assertEquals("Error : the filter doesn't have the operator Equals ", equals.getName().getLocalPart(), "Equals");
 
         //test Disjoint
-        xml = OGC_TestFilterEncoding.class.getResourceAsStream("filter_Disjoint.xml");
+        xml = OGC_FilterEncodingTest.class.getResourceAsStream("filter_Disjoint.xml");
```

Comment:
The travis check fails because the resource file `filter_Disjoint.xml` is called `Filter_Disjoint.xml` with an upper case **F**